### ISSUE TITLE
Duck.gltf does not displayed by filament v1.3.0

### ISSUE THREAD

**Issue by cx20** [google/filament#1430]
**Describe the bug**
I tried to create a display sample of [glTF model](https://github.com/cx20/gltf-test/tree/master/sampleModels/Duck/glTF) by filament with [gltf-test](https://github.com/cx20/gltf-test).

**To Reproduce**
[glTF/Duck.gltf + filament result:](https://cx20.github.io/gltf-test/examples/filament/index.html?model=Duck&scale=1&type=glTF) NG
The following error is displayed on the console:
```
filament.js:6  GET https://cx20.github.io/gltf-test/examples/filament/Duck0.bin 404
filament.js:6  GET https://cx20.github.io/gltf-test/examples/filament/DuckCM.png 404
```
It seems that `.bin` and `.png` files point to different places. Correctly, it is in the following place.
/gltf-test/sampleModels/Duck/glTF/Duck.gltf
/gltf-test/sampleModels/Duck/glTF/Duck0.bin
/gltf-test/sampleModels/Duck/glTF/DuckCM.png

**Expected behavior**
The `glTF-Embedded` format and the `glTF-Binary` format are displayed correctly. It does not appear correctly if the resources are split.

[glTF-Embedded/Duck.gltf + filament result:](https://cx20.github.io/gltf-test/examples/filament/index.html?model=Duck&scale=1&type=glTF-Embedded) OK
[glTF-Binary/Duck.glb + filament result:](https://cx20.github.io/gltf-test/examples/filament/index.html?model=Duck&scale=1&type=glTF-Binary) OK
![image](https://user-images.githubusercontent.com/1833979/61591557-0b8a4080-ac03-11e9-81a5-ddbddb6fa18f.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 + Chrome 75
 - GPU: Intel HD Graphics 520
 - Backend: WebGL/ANGLE (Intel(R) HD Graphics 520 Direct3D11 vs_5_0 ps_5_0)

**Comment by romainguy** [comment]
Thank you! I believe this has already been fixed in the master branch. @prideout can confirm.

**Comment by prideout** [comment]
@cx20, what is the basePath that is being passed to `asset.loadResources`?  It needs to be the same as the resource folder.

**Comment by cx20** [comment]
Oops, In v1.3.0, the third argument of loadResources was not implemented yet.

![image](https://user-images.githubusercontent.com/1833979/61596660-6c386e00-ac41-11e9-97e6-39854526ec80.png)

**Comment by cx20** [comment]
I used [filament.js](https://github.com/google/filament/blob/master/docs/webgl/filament.js) (2019/07/12 4:18 JST) in the following location.
https://github.com/google/filament/blob/master/docs/webgl/filament.js
However, in this version, the `basePath` argument of the `loadResources` function does not seem to be implemented yet.

**Comment by prideout** [comment]
Yes, the latest npm release (as well as our web site) is a few weeks behind head.

**Comment by cx20** [comment]
I understand. I will try again when the basePath argument is available.

**Comment by cx20** [comment]
I confirmed that the problem was solved with filament v1.3.1.
[glTF/Duck.gltf + filament result](https://cx20.github.io/gltf-test/examples/filament/index.html?model=Duck&scale=1&type=glTF): OK
![image](https://user-images.githubusercontent.com/1833979/62871623-cc10c900-bd56-11e9-8226-353ce118c8c4.png)
